### PR TITLE
SFT-81 Update song downloader.

### PR DESCRIPTION
**Pull request by DatLam25**
1. Unnecessary codes are removed from the original script. 
2. Create new script that automatically download all the songs from the AWS bucket down to local.
3. Update README with instructions on how to download the songs.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,3 +9,14 @@ The media player will search for songs in the `/home/{user}/Music/SongLibrary` f
 ## Class Diagram
 
 ![Class Diagram](pictures/ClassDiagram.png)
+
+## Download All Song
+1. Installing AWSCLI
+In order to download all song, first you need to install AWSCLI by running:
+`sudo apt-get install awscli`
+
+2. Obtaining Acess Keys
```
Comment (bill-luu) [inline README.md]: Misspelled `Access`

File: README.md [hunk 2]
```diff
@@ -9,3 +9,14 @@ The media player will search for songs in the `/home/{user}/Music/SongLibrary` f
...
+To directly download from the aws bucket, you will need to obtain the access key ID and the secret acess key.
```
Comment (bill-luu) [inline README.md]: `secret access key`

File: README.md [hunk 3]
```diff
@@ -9,3 +9,14 @@ The media player will search for songs in the `/home/{user}/Music/SongLibrary` f
...
+
+3. Downloading all songs
+With the access keys, you can now run `./DownloadAllSong.sh` and inputting the acess keys into the apporiate places. The region name is `us-west-2`.
```
Comment (bill-luu) [inline README.md]: `acess` :)

Comment (bill-luu) [reply README.md]: `appropriate`

**Review by bill-luu — CHANGES_REQUESTED** [review]

**Comment by bill-luu** [comment]
Is there any point to the original script if the new one works?

**Comment by mglyngberg** [comment]
> 
> 
> Is there any point to the original script if the new one works?

I can see value if someone only wants to test the music player with a couple of songs instead of having to download all of them. It might be better to add a flag or something to the new script instead of having two though.

File: DownloadSongs.sh [hunk 4]
```diff
@@ -3,11 +3,22 @@
 mkdir -p $HOME/Music/SongLibrary
 cd $HOME/Music/SongLibrary
 
+while getopts ":a" opt; do
+  case $opt in
+    a)
+	aws configure
```
Comment (bill-luu) [inline DownloadSongs.sh]: We don't need to configure every time we run the script, it makes more sense to me that this step should be done manually, described in the readme in step 2.

File: DownloadSongs.sh [hunk 5]
```diff
@@ -3,11 +3,22 @@
...
+
 song=(1-06+Animal 1-11+Armageddon+It 1-14+Hysteria 1-01+Good+Times+Bad+Times 20+Rock+and+Roll+Is+King)
 
 for i in "${song[@]}"; do
 	if [ ! -f $i.mp3 ]; then
 	wget https://s3-us-west-2.amazonaws.com/ucsolarteam.onboardmedia.songs/$i.mp3
 	fi
-done 
-
+done
```
Comment (bill-luu) [inline DownloadSongs.sh]: If we run the script with `-a`, there's no point in downloading these songs again

**Review by bill-luu — CHANGES_REQUESTED** [review]

File: README.md [hunk 6]
```diff
@@ -9,3 +9,24 @@ The media player will search for songs in the `/home/{user}/Music/SongLibrary` f
...
+
+In order to download all song, you need to install AWS CLI by running `sudo apt-get install awscli`
+
+**2. Obtain Access Keys**
+
+To directly download from the aws bucket, you will need to obtain the Access Key ID and the Secret Access Key.
+
+**3. Configure the AWS CLI**
+
+Run `aws configure` in the terminal to configure the profile for AWS. Input the access keys obtained. The region name is `us-west-2`.
```
Comment (bill-luu) [inline README.md]: Seperate sentences so it's one line each, it's easier to read and still shows up in the rich text view as one paragraph.
```
Run `aws configure` in the terminal to configure the profile for AWS.
Input the access keys obtained.
The region name is `us-west-2`.
```

File: README.md [hunk 7]
```diff
@@ -9,3 +9,24 @@ The media player will search for songs in the `/home/{user}/Music/SongLibrary` f
 ## Class Diagram
 
 ![Class Diagram](pictures/ClassDiagram.png)
+
+## Download Songs
+If you only need a small number of song, you can run the `./DownloadSongs.sh` in the terminal to download a pre-selected selection of songs.
```
Comment (bill-luu) [inline README.md]: `a small number of song`->`a small number of songs`

**Review by bill-luu — CHANGES_REQUESTED** [review]

**Review by bill-luu — APPROVED** [review]

**Pull request merged**